org.knime.knip.io/src/org/knime/knip/io/nodes/imgreader2/readfrominput/ImgReaderTableNodeModel.java

```diff
@@ -201,7 +200,7 @@ public void runFinal(PortInput[] inputs, PortOutput[] outputs, ExecutionContext
 					readImgFunction.apply(row).forEachOrdered(result -> {
 						if (result.getSecond().isPresent()) {
 							encounteredExceptions.set(true);
-							LOGGER.debug("Encountered exception while reading image " + result.getFirst().getKey()
+							LOGGER.warn("Encountered exception while reading image " + result.getFirst().getKey()
```

Comment:
Actually I've set them to debug on purpose. You don't want to get spammed with warnings. You can look at the log if you want more information.